src/Native/Runtime/RhConfig.cpp

```diff
@@ -219,15 +219,19 @@ void RhConfig::ReadConfigIni()
 }
 
 //returns the path to the runtime configuration ini
-_Ret_maybenull_z_ WCHAR* RhConfig::GetConfigPath()
+_Ret_maybenull_z_ TCHAR* RhConfig::GetConfigPath()
 {
-
-    WCHAR* exePathBuff;
+    TCHAR* exePathBuff;
 
     //get the path to rhconfig.ini, this file is expected to live along side the app 
     //to build the path get the process executable module full path strip off the file name and 
     //append rhconfig.ini
+#ifdef PLATFORM_UNIX
+    // UNIXTODO: Implement RhConfig::GetConfigPath!
```

Comment:
There will be more changes needed as part of PalGetModuleFileName Unix port, so I have not bothered with it.